src/main/resources/cli_en_US.properties

```
@@ -40,10 +40,23 @@ uninstall.error.identifier_not_defined=You must specify plug-in identifier when
 uninstall.error.plugin_not_found=Plug-in %s not found
 # Transtypes subcommand
 transtypes.usage=dita transtypes [options]
+# Init subcommand
+init.usage=dita init <template> [options]
+init.argument.template=Template name
+init.option.output=Output directory
+init.option.list=List installed templates
+init.error.template_not_defined=Template must be defined
+init.error.file_already_exists=Failed to create project, file already exists: %s
+init.error.create_failed=Failed to create project: %s
+init.error.template_not_found=Template %s not found
+init.error.list_failed=Failed to list templates
+init.info.create=Init project using %s template to %s
```

Let's spell this out and quote the placeholders as we do in other messages since https://github.com/dita-ot/dita-ot/commit/9736df0:

```suggestion
init.info.create=Initialized new project in '%s' using the '%s' template
```

2 questions:
1. Do the quotes need to be escaped in `.properties` files?
2. Is it OK to flip the order of arguments here, or would that require other changes?

---

Quotes are ok. If we want to switch the order, that requires code changes. What should the new copy be?